popup/popup.css

```
@@ -31,16 +33,41 @@ button {
   width: 100%;
+  transition: all 0.2s ease;
+  transform: translateY(0);
   margin-bottom: 8px;
   font-weight: 500;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  /* gap: 8px; - Reverting to margin for reliability */
+  text-transform: uppercase;
+  letter-spacing: 0.5px;
+  font-size: 13px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)